dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -220,30 +220,30 @@ STRINGTABLE
 BEGIN
     IDS_LOGGEDOUTSAS "Bun venit!"
     IDS_LOCKEDSAS "Momentan calculatorul este blocat."
-    IDS_PRESSCTRLALTDELETE "Pentru autentificare, apăsați «Ctrl»+«Alt»+«Del»."
+    IDS_PRESSCTRLALTDELETE "Pentru Log on, apăsați Ctrl-Alt-Delete."
     IDS_ASKFORUSER "Nume utilizator: "
     IDS_ASKFORPASSWORD "Parola: "
     IDS_FORCELOGOFF "Această acțiune va deautentifica utilizatorului curent cu posibile pierderi ale datelor nesalvate. Continuați?"
-    IDS_LOCKMSG "Doar %s sau un Administrator poate debloca acest calculator."
+    IDS_LOCKMSG "Numai %s\\%s (%s) sau un administrator au posibilitatea să deblocheze acest computer."
```

Comment:
Post a screenshot of that line is en-US! Unless it has 3 %s as well, this might be a bug.

Reply:
This is for en-US

![Windows XP Professional-2024-03-11-15-48-07](https://github.com/reactos/reactos/assets/113553683/40e97b05-f68f-48d8-8526-3565431dc95b)